package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "console": "node console.js",
     "cover": "npm run test -- --coverage",
-    "test": "NODE_ENV=testing jest --runInBand --forceExit --detectOpenHandles",
```

Comment:
Don't modify this

Reply:
Because i'm getting the open handles warning and it looks like the test are failing but they are not